insta-comments-scraper/comments-scraper.go

```diff
@@ -136,13 +131,56 @@ func (p *PostCommentScraper) sendComments(postsComments *models.InstaPostComment
 	return p.commentsInfoQWriter.WriteMessages(context.Background(), messages...)
 }
 
+func (p *PostCommentScraper) scrapePostComments(shortCode string) (models.InstaPostComments, error) {
+	var instaPostComment models.InstaPostComments
+	type Variables struct {
+		Shortcode           string `json:"shortcode"`
+		ChildCommentCount   int    `json:"child_comment_count"`
+		FetchCommentCount   int    `json:"fetch_comment_count"`
+		ParentCommentCount  int    `json:"parent_comment_count"`
+		HasThreadedComments bool   `json:"has_threaded_comments"`
+	}
+
+	variable := &Variables{shortCode, 3, 40, 24, true}
+	variableJson, err := json.Marshal(variable)
+	if err != nil {
+		return instaPostComment, err
+	}
+
+	queryEncoded := url.QueryEscape(string(variableJson))
+	url := fmt.Sprintf(userPostsCommentUrl, queryEncoded)
+
+	request, err := http.NewRequest("GET", url, nil)
+
+	if err != nil {
+		return instaPostComment, err
+	}
+	p.httpClient.AddHeaders(request)
+	response, err := p.httpClient.GetClient().Do(request)
```

Comment:
move the `AddHeaders(request)` call inside the `httpClient` and instead of calling `GetClient().Do` just add `Do` to the `ScraperClient` interface